cmd/aggregated-apiserver/app/options/options.go

```diff
@@ -39,19 +42,50 @@ import (
 	"k8s.io/client-go/util/flowcontrol"
 	"k8s.io/klog/v2"
 	netutils "k8s.io/utils/net"
+	ctrlmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
```

Comment:
@XiShanYongYe-Chang 
Please confirm if it is appropriate to use the registry from `controller-runtime`, I'm not sure if there is any metric registered to Kubernetes' registry.

Reply:
Fixed

Reply:
Thanks @XiShanYongYe-Chang 
`controller-runtime metrics` is more in line with my original intention of adding it.

@RainbowMango  wdyt?

Reply:
Both OK to me, depends on you, given Kubernetes doesn't provide other necessary metrics.